public/app/features/explore/ExploreTimeControls.tsx

```diff
@@ -85,7 +86,7 @@ export class ExploreTimeControls extends Component<Props> {
       onZoom: this.onZoom,
       hideText,
     };
-
+    const weekStart = getWeekStart(config.bootData.user.weekStart);
```

Comment:
@Clarity-89 this will not work when user updates the preferences, can't we use the current locale? 

https://github.com/grafana/grafana/blob/224e96effa3dcae21311a94dc3348821862367b1/packages/grafana-data/src/datetime/moment_wrapper.ts#L158